packages/suite/src/actions/wallet/coinmarket/coinmarketCommonActions.ts

```diff
@@ -186,3 +195,41 @@ export const setLoading = (
 export const loadInvityData = (): CoinmarketCommonAction => ({
     type: COINMARKET_COMMON.LOAD_DATA,
 });
+
+export const convertDrafts = () => (dispatch: Dispatch, getState: GetState) => {
+    const { accounts, formDrafts, settings } = getState().wallet;
+    const formDraftKeys = Object.keys(formDrafts);
+
+    formDraftKeys.forEach(formDraftKey => {
```

Comment:
Since you are not checking for the route here, the action is fired more times than it should. Or is there a reason to for it?

Reply:
Yep, that's possible. The idea was for the code to be generic for Buy, Sell and Exchange.

Reply:
Ok, let's hope there won't be any issues. it's not that detrimental. What do you think? @matejkriz